README.md

```diff
@@ -4,12 +4,17 @@
 This repository contains a java project that converts RAML v0.8 API definition to Swagger v2.0 definition. 
```

Comment:
Could you please mention here that the project could be used both as a standalone tool and also as a lib? I am travelling. So, I couldn't do it now.

Reply:
Sure will do.

Reply:
Thats really good work. Also, I would like you to send the pull request to the new "Update" branch which I created now.